testutils/clustermanager/perf-tests/pkg/cluster.go

```diff
@@ -230,11 +230,14 @@ func (gc *gkeClient) createClusterWithRetries(gcpProject, name string, config Cl
 		addons = strings.Split(config.Addons, ",")
 	}
 	req := &gke.Request{
+		Project:     gcpProject,
 		ClusterName: name,
 		MinNodes:    config.NodeCount,
 		MaxNodes:    config.NodeCount,
 		NodeType:    config.NodeType,
 		Addons:      addons,
+		// Always enable Workload Identity.
```

Comment:
Can you use something like "Enable Workload Identity for perf tests because ...blabala..." so it's clear why we need this for perf clusters?

Reply:
Added some comments.
But since this is only used in internal tests, I didn't add much detail.